sr_utilities/scripts/sr_utilities/hand_finder.py

```diff
@@ -92,7 +92,7 @@ def get_default_joints(cls):
                   'THJ5', 'WRJ1', 'WRJ2']
         return joints
 
-    def __init__(self, mapping, joint_prefix):
+    def __init__(self, mapping, joint_prefix, hand_h=None):
```

Comment:
That's correct. It's not used anywhere that would currently be used by a Hand H... It would likely be a useful thing at some point but def not necessary right now. Can add it if you want, just figured to save time for now...